src/app/backend/apihandler.go

```diff
@@ -50,6 +50,17 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 			Writes(ReplicaSetDetail{}))
 	wsContainer.Add(replicaSetWs)
 
+	namespaceCreationWs := new(restful.WebService)
+	namespaceCreationWs.Path("/api/namespacecreation").
+		Consumes(restful.MIME_JSON).
+		Produces(restful.MIME_JSON)
+	namespaceCreationWs.Route(
+		namespaceCreationWs.POST("").
```

Comment:
To create a namespace you should use existing web service on path "/api/namespaces" and handle POST method there. This is how RESTful apis should work. Initial reading on this: https://en.wikipedia.org/wiki/Representational_state_transfer